bumble/smp.py

```diff
@@ -525,6 +525,9 @@ async def compare_numbers(self, number: int, digits: int) -> bool:
     async def get_number(self) -> int:
         return 0
 
+    async def get_string(self) -> str:
```

Comment:
I suggest adding a `max_length` argument to let the delegate know what's the maximum number of encoded bytes that are expected. This can be useful for a user interface to limit the text input field.

Reply:
I will add it